content/docs/guides/components/hyperparameter.md

```diff
@@ -34,13 +34,110 @@ kubectl get pod -n kubeflow  # Find your vizier-core and modedb-frontend pods
 kubectl port-forward -n kubeflow [vizier-core pod] 6789:6789 &
 kubectl port-forward -n kubeflow [modeldb-frontend pod] 3000:3000 &
 ```
+## Creating a Study Job
+You can create Study Job for Katib by defining a StudyJob config file.
+See [random algorithm example](https://github.com/kubeflow/katib/blob/master/examples/random-example.yaml).
 
-Run the client [script](https://github.com/kubeflow/katib/blob/master/examples/client-example.go)
 ```
-go run client-example.go
+$ kubectl apply -f random-example.yaml
+```
+Only this command, a study will start, generate hyper-parameters and save the results.
```

Comment:
Only this command, a study will start, generate hyper-parameters and save the results. -> 
"Running this command will launch a StudyJob. The study job will run a series of training jobs to train models using different hyperparameters and save the results."